native/SampleApps/SmartSyncExplorer/SmartSyncExplorer/Classes/AppDelegate.m

```diff
@@ -73,9 +73,8 @@ - (id)init
         
         //Uncomment following block to enable IDP Login flow.
         /*
-        [SalesforceSDKManager sharedManager].idpEnabled = YES;
          //scheme of idpAppp
-        [SalesforceSDKManager sharedManager].idpAppScheme = @"sampleidpapp";
+        [SalesforceSDKManager sharedManager].idpAppURIScheme = @"sampleidpapp";
```

Comment:
Isn't this code commented out in the other sample apps' `AppDelegate` too? If not, we should throw it in there too.

Reply:
Sure can.